plugins/Marketplace/lang/en.json

```diff
@@ -5,78 +5,78 @@
+        "NoticeRemoveMarketplaceFromReportingMenu": "Remove the Marketplace from the reporting menu by installing %1$sthe White Label%2$s plugin.",
         "Marketplace": "Marketplace",
         "PaidPlugins": "Premium Features",
         "FeaturedPlugin": "Featured plugin",
-        "InstallingNewPluginsViaMarketplaceOrUpload": "You may automatically install plugins from the Marketplace or %1$supload a plugin%2$s in .zip format.",
-        "InstallingNewThemesViaMarketplaceOrUpload": "You may automatically install themes from the Marketplace or %1$supload a theme%2$s in .zip format.",
-        "InstallingPlugin": "Installing %s",
+        "InstallingNewPluginsViaMarketplaceOrUpload": "Auto-install plugins from the Marketplace or %1$supload a plugin%2$s in .zip format.",
+        "InstallingNewThemesViaMarketplaceOrUpload": "Auto-install themes from the Marketplace or %1$supload a theme%2$s in .zip format.",
+        "InstallingPlugin": "Installing %s…",
         "InstallPurchasedPlugins": "Install purchased plugins",
         "LastCommitTime": "(last commit %s)",
         "LastUpdated": "Last Updated",
         "License": "License",
         "LicenseKey": "License key",
-        "ManageLicenseKeyIntro": "You can extend Matomo's capabilities using our premium plugins. To install premium plugins, you need a valid license key, which can be obtained in two ways: you can start a trial for your selected plugin in the %1$smarketplace%2$s, or purchase a plugin subscription on %3$sour website%4$s. Starting a trial in the marketplace will automatically create a license key and add it to this instance. If you purchase a subscription on the website, you will receive a license key that you can add it in the field below. After your license key is added, you can update or remove it if needed.",
-        "LicenseKeyActivatedSuccess": "License key successfully activated!",
-        "LicenseKeyDeletedSuccess": "License key successfully deleted.",
+        "ManageLicenseKeyIntro": "Try a premium plugin from %1$sthe marketplace%2$s, or get a license on %3$sthe website%4$s first and add it below.",
+        "LicenseKeyActivatedSuccess": "License key activated.",
+        "LicenseKeyDeletedSuccess": "License key deleted.",
         "Exceeded": "Exceeded",
         "LicenseMissing": "License missing",
-        "LicenseMissingDeactivatedDescription": "The following plugins have been deactivated because you are using them without a license: %1$s. %2$sTo resolve this issue either update your license key, %3$sget a subscription now%4$s or deactivate the plugin.",
-        "PluginLicenseMissingDescription": "You are not allowed to download this plugin because there is no license for this plugin. To resolve this issue either update your license key, get a subscription or uninstall the plugin.",
+        "LicenseMissingDeactivatedDescription": "%2$sRenew your license to keep using %1$s, or %3$sget a new license%4$s.",
```

Comment:
I don't understand what %2$s is doing here.

Reply:
It adds a `<br />`